mobile/lib/modules/home/ui/thumbnail_image.dart

```diff
@@ -60,25 +62,14 @@ class ThumbnailImage extends HookConsumerWidget {
               .watch(homePageStateProvider.notifier)
               .addSingleSelectedItem(asset);
         } else {
-          if (asset.type == AssetTypeEnum.IMAGE) {
-            AutoRouter.of(context).push(
-              ImageViewerRoute(
-                imageUrl:
-                    '${box.get(serverEndpointKey)}/asset/file?aid=${asset.deviceAssetId}&did=${asset.deviceId}&isThumb=false',
-                heroTag: asset.id,
-                thumbnailUrl: thumbnailRequestUrl,
-                asset: asset,
-              ),
-            );
-          } else {
-            AutoRouter.of(context).push(
-              VideoViewerRoute(
-                videoUrl:
-                    '${box.get(serverEndpointKey)}/asset/file?aid=${asset.deviceAssetId}&did=${asset.deviceId}',
-                asset: asset,
-              ),
-            );
-          }
+          AutoRouter.of(context).push(
+            GalleryViewerRoute(
+              assetList: assetList,
+              box: box,

```

Comment:
Since box can be open anywhere, I don't think we need to make it as an argument of the Widget

Reply:
Got it, will remove.